Source/Core/TaskProcessor.js

```diff
@@ -6,8 +6,10 @@ define([
         './destroyObject',
```

Comment:
Even though Draco doesn't work in IE yet I went to open Sandcastle in IE and it doesn't get past the loading screen. `master` seems fine. Is this a result of any of the changes here?

Reply:
Merged in master, that should incorporate the error message fix.